test/lib/pgBackRestTest/Common/RunTest.pm

```diff
@@ -300,7 +247,6 @@ sub testResult
             {name => 'strExpected', required => false, trace => true},
             {name => 'strDescription', trace => true},
             {name => 'iWaitSeconds', optional => true, default => 0, trace => true},
-            {name => 'strLogExpect', optional => true, trace => true},
             {name => 'strLogLevel', optional => true, default => WARN, trace => true},
```

Comment:
mismatch strLogLevel exists on rhs of assignment but not left???

Reply:
Fixed in 3e8942c47013d9. Interesting that Perl was cool with this.

Reply:
Ah, I see. Perl is fine with the lvalue array being smaller than the rvalue array. Might be fine the other way, too.

Reply:
Interesting. I wasn't aware of that. Previous jobs had some perl portions, but not a lot of it overlapped with what I mainly worked on.